packages/cloudflare/src/cli/build/build-worker.ts

```diff
@@ -1,4 +1,4 @@
-import { Config } from "../config";
```

Comment:
l 50 should be updated with the new path?

Reply:
good point!

it currently works as intended but that's because we get a wrong "packageDir" ([here](https://github.com/opennextjs/opennextjs-cloudflare/pull/31/files#diff-3806ec13864197c25b18c92269913af6a5852467e9c476a4d46f1d3430192954L20)):
![Screenshot 2024-09-25 at 12 49 08](https://github.com/user-attachments/assets/3984df57-510d-4525-aa48-4745a4cf9621)

I'll update this to make it correct

Reply:
That should help as I will need to ref `getRequestContext()` in the cache handler